docs/spec/relayer/PacketHandlers.tla

```diff
@@ -0,0 +1,123 @@
+--------------------------- MODULE PacketHandlers ---------------------------
+
+(***************************************************************************
+ This module contains definitions of operators that are used to handle
+ packet datagrams
+ ***************************************************************************)
+
+EXTENDS Integers, FiniteSets, RelayerDefinitions    
+
+(***************************************************************************
+ Packet datagram handlers
+ ***************************************************************************)
+
+\* Handle "PacketRecv" datagrams
+HandlePacketRecv(chainID, chain, packetDatagram, log) ==
+    \* get chainID's connection end
+    LET connectionEnd == chain.connectionEnd IN
```

Comment:
I am wondering if we are missing important aspect here as we don't use chain and connection id from packetDatagram to fetch local connection and channel end.